src/mixins.scss

```
@@ -29,3 +29,19 @@
     gap: 70px;
   }
 }
+
+@mixin rows-with-gap($gap) {
+  display: flex;
+  gap: $gap;
+}
+
+@mixin columns-with-gap($gap) {
+  display: flex;
+  flex-direction: column;
+  gap: $gap;
+}
```

To be more understandable at first sight, I would have added the `flex` notion in naming:
```suggestion
@mixin flex-rows-with-gap($gap) {
  display: flex;
  gap: $gap;
}

@mixin columns-with-gap($gap) {
  display: flex;
  flex-direction: column;
  gap: $gap;
}
```